### PR TITLE
Enable new EU countries on VPN landing page (Fixes #10280)

### LINKED ISSUE #10280: Add support for new EU countries to VPN landing page
## Description

Country list and exact launch date is still TBD. 

Note: In some of these new countries there may not be a single national language, and instead of mix of two (or more) different languages that are commonly spoken. In these cases we'll need to add some new functionality to the landing page, whereby certain countries can use alternate subscription plan IDs based on the locale of the page. We still need to finalize exactly how this will work with the sub-plat team.

---

#### Success Criteria
- [x] VPN landing page supports additional countries for variable pricing plans.
- [x] New countries are behind a feature switch.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
Enables VPN availability in new EU countries (behind a feature switch)

## Issue / Bugzilla link
#10280

## Testing
Note: this will go through a week of testing by the product QA team prior to launch on www-dev, but I tried to provide some (hopefully) easy to follow steps for reviewing the changes here:

To test this PR please set the following in your `.env` file:

```
SWITCH_VPN_VARIABLE_PRICING_WAVE_1=off
SWITCH_VPN_VARIABLE_PRICING_EU_EXPANSION=on
```
**New countries:**

Page should display variable pricing in Euros:

- [x] http://localhost:8000/en-US/products/vpn/?geo=at (Austria)
- [x] http://localhost:8000/en-US/products/vpn/?geo=be (Belgium)
- [x] http://localhost:8000/en-US/products/vpn/?geo=es (Spain)
- [x] http://localhost:8000/en-US/products/vpn/?geo=it (Italy)

Page should display variable pricing in Swis Francs:

- [x] http://localhost:8000/en-US/products/vpn/?geo=ch (Switzerland)

**Not available:**

- [x] http://localhost:8000/en-US/products/vpn/?geo=pl
   - [x] Page should mention "Available in 13 countries now".
   - [x] Page copy should reference the new additions: Italy, Spain, Belgium, Austria, Switzerland.

### CODE REVIEW

**Comment by alexgibson** [comment]
@pmac @stephaniehobson this isn't ready to merge just yet (we're waiting on the production IDs), but should be ready for code review? I tried to add some (hopefully) easy enough to follow testing instructions here in the PR, but I'd also be happy to talk through the changes if that might be useful. Let me know?

**Comment by alexgibson** [comment]
This is ready and needs to merge by Friday 25th (so QA can test on www-dev during the wellness week)

File: bedrock/products/templatetags/misc.py [hunk 1]
```diff
@@ -80,13 +80,19 @@ def vpn_subscribe_link(ctx, entrypoint, link_text, plan=None, class_name=None, l
     if plan in ['12-month', '6-month', 'monthly']:
 
         # Set a default plan ID using page locale. This acts as a fallback should geo-location fail.
-        pricing_params = settings.VPN_VARIABLE_PRICING.get(lang, settings.VPN_VARIABLE_PRICING['us'])
-        plan_default = pricing_params[plan]['id']
+        cc = lang.split('-')[1].lower() if lang and '-' in lang else lang
```
Comment (pmac) [inline bedrock/products/templatetags/misc.py:83]: should the last part be `lang.lower()` or will the langs with no dash in them definitely be lower case?

File: bedrock/products/templatetags/misc.py [hunk 2]
```diff
@@ -117,16 +123,19 @@ def vpn_monthly_price(ctx, plan='monthly', lang=None):
         {{ vpn_monthly_price(plan='12-month') }}
     """
 
-    pricing_params = settings.VPN_VARIABLE_PRICING.get(lang, settings.VPN_VARIABLE_PRICING['us'])
-    default_amount = pricing_params[plan]['price']
-    euro_amount = settings.VPN_VARIABLE_PRICING['de'][plan]['price']
-    usd_amount = settings.VPN_VARIABLE_PRICING['us'][plan]['price']
+    cc = lang.split('-')[1].lower() if lang and '-' in lang else lang
+    default_params = settings.VPN_VARIABLE_PRICING.get(cc, settings.VPN_VARIABLE_PRICING['us'])
```
Comment (pmac) [inline bedrock/products/templatetags/misc.py:127]: Perhaps this should be a `get_default_params(lang)` helper function?

**Review by pmac — APPROVED** [review]
A couple of notes, but overall looks excellent. 🍰

**Review by stephaniehobson — APPROVED** [review]
R+ 👑 

This is pretty complicated but the tests are excellent and all the stated test cases and the edge cases I tried myself returned the expected results. I'm comfortable giving this an R+ without a walk through, but if you want someone to double check the logic we can go through it together.

File: bedrock/products/templatetags/misc.py (hunk 1 above)
Comment (alexgibson) [reply bedrock/products/templatetags/misc.py:83]: Worth adding just to be extra safe 👍

File: bedrock/products/templatetags/misc.py (hunk 2 above)
Comment (alexgibson) [reply bedrock/products/templatetags/misc.py:127]: Good suggestion!

**Comment by alexgibson** [comment]
Thanks for the review @pmac and @stephaniehobson 👍 

I added a followup commit to address pmac's comments. This is going to be QA'd next week on staging, so I think we're OK not going into the finer details too much on all the pricing plan logic. In the future the product /subscription teams want to find a way to simplify things (see https://mozilla-hub.atlassian.net/browse/PSP-42)

If all looks well, I think we're good to merge here?

**Comment by alexgibson** [comment]
Ok, I'm gonna merge this so QA can get a head-start - thanks both!
